api/jobs.yaml

```diff
@@ -116,6 +131,40 @@ paths:
       tags:
         - Jobs
 definitions:
+  AuthenticationCapability:
+    description: Specification of the backend's authentication capabilities
+    properties:
+      is_required:
+        type: boolean
+        description: Whether or not authentication is required
+      scopes:
+        type: array
+        description: OAuth 2.0 requested scopes
+        items:
+          type: string
+  CapabilitiesResponse:
+    description: Specification of the backend's capabilities
+    properties:
+      authentication:
+        $ref: '#/definitions/AuthenticationCapability'
+      display_fields:
+        type: object
+        description: >
+          Fields on QueryJobsResult returned from POST /jobs/query
+          populated on some or all jobs. The fields are mapped to their 
+          display names, in order of importance. Extended fields and
+          labels can be included, such as 'label.foo' or
+          'extended_fields.user_id'
+      label_query_fields:
```

Comment:
Per comment I just added to the design doc, I think this naming should be generalized and description updated.

Reply:
Done.